src/cli/rustup_mode.rs

```diff
@@ -548,11 +548,10 @@ pub async fn main(current_dir: PathBuf, process: &Process) -> Result<utils::Exit
             info!("This is the version for the rustup toolchain manager, not the rustc compiler.");
             let mut cfg = common::set_globals(current_dir, false, true, process)?;
             match cfg.active_rustc_version() {
-                Ok(Some(version)) => info!("The currently active `rustc` version is `{}`", version),
+                Ok(Some(version)) => info!("The currently active `rustc` version is `{version}`"),
                 Ok(None) => info!("No `rustc` is currently active"),
                 Err(err) => trace!(
-                    "Wanted to tell you the current rustc version, too, but ran into this error: {}",
-                    err
+                    "Wanted to tell you the current `rustc` version, but ran into this error: {err}",
```

Comment:
Since you're changing this, maybe also make this a little less familiar? Like "Wanted to tell you the current rustc version" -> "Failed to display the current rustc version"?

Reply:
@djc Makes sense. There's a certain amount of "fun" in the original wording and it's a bit hard to determine how much of that "fun" we'd like to have in our project...

Reply:
Done.